control-plane/controllers/resources/mesh_gateway_controller_test.go

```diff
@@ -444,7 +586,7 @@ func TestMeshGatewayController_Reconcile(t *testing.T) {
 
 			res, err := controller.Reconcile(context.Background(), testCase.request)
 			if testCase.expectedErr != nil {
-				//require.EqualError(t, err, testCase.expectedErr.Error())
+				// require.EqualError(t, err, testCase.expectedErr.Error())
```

Comment:
Should this be here? Out of scope for this PR, just curious

Reply:
honestly I don't know, this was gofumpt editing the spacing on the comment